tests/wallet/test_wallet.py

```diff
@@ -762,95 +761,190 @@ async def test_clawback_resync(
+        )
+        before_txs["sender"][
+            TransactionType.COINBASE_REWARD
+        ] = await wallet_node_1.wallet_state_manager.tx_store.get_transaction_count_for_wallet(
+            1, type_filter=TransactionTypeFilter.include([TransactionType.COINBASE_REWARD])
+        )
+        before_txs["recipient"][
+            TransactionType.INCOMING_CLAWBACK_RECEIVE
+        ] = await wallet_node_2.wallet_state_manager.tx_store.get_transaction_count_for_wallet(
+            1, type_filter=TransactionTypeFilter.include([TransactionType.INCOMING_CLAWBACK_RECEIVE])
+        )
+        # Resync start
+        wallet_node_1._close()
+        await wallet_node_1._await_closed()
         wallet_node_2._close()
         await wallet_node_2._await_closed()
-        # set flag to reset wallet sync data on start
-        await api_0.set_wallet_resync_on_startup({"enable": True})
-        fingerprint = wallet_node.logged_in_fingerprint
-        assert wallet_node._wallet_state_manager
-        # 2 reward coins, 1 clawbacked coin
-        assert len(await wallet_node._wallet_state_manager.coin_store.get_all_unspent_coins()) == 7
-        # standard wallet
-        assert len(await wallet_node.wallet_state_manager.user_store.get_all_wallet_info_entries()) == 1
-        before_txs = await wallet_node.wallet_state_manager.tx_store.get_all_transactions()
-        # Delete tx records
-        await wallet_node.wallet_state_manager.tx_store.rollback_to_block(0)
-        wallet_node._close()
-        await wallet_node._await_closed()
-        config = load_config(wallet_node.root_path, "config.yaml")
-        # check that flag was set in config file
-        assert config["wallet"]["reset_sync_for_fingerprint"] == fingerprint
-        new_config = wallet_node.config.copy()
-        new_config["reset_sync_for_fingerprint"] = config["wallet"]["reset_sync_for_fingerprint"]
-        new_config["database_path"] = "wallet/db/blockchain_wallet_v2_test_CHALLENGE_KEY.sqlite"
-        wallet_node_2.config = new_config
-        wallet_node_2.root_path = wallet_node.root_path
-        wallet_node_2.local_keychain = wallet_node.local_keychain
+        wallet_node_1.config["database_path"] = "wallet/db/blockchain_wallet_v2_test1_CHALLENGE_KEY.sqlite"
+        wallet_node_2.config["database_path"] = "wallet/db/blockchain_wallet_v2_test2_CHALLENGE_KEY.sqlite"
+
         # use second node to start the same wallet, reusing config and db
-        await wallet_node_2._start_with_fingerprint(fingerprint)
-        assert wallet_node_2._wallet_state_manager
-        await server_3.start_client(PeerInfo(self_hostname, uint16(server_1._port)), None)
+        await wallet_node_1._start()
+        assert wallet_node_1.wallet_state_manager is not None
+        await wallet_server_1.start_client(PeerInfo(self_hostname, uint16(full_node_server._port)), None)
+        await wallet_node_2._start()
+        assert wallet_node_2.wallet_state_manager is not None
```

Comment:
```suggestion
```

Reply:
Resolving per discussion with @xdustinface and @ytx1991